test/basic-querying.test.js

```diff
@@ -620,6 +620,61 @@ describe('basic-querying', function () {
   });
 });
 
+describe('id field', function() {
+  var Todo = getSchema().define('Todo', {
+    id: {type: 'number', id: false},
```

Comment:
@raymondfeng Does this look correct? I wasn't sure how to set the `id: false` here.